src/region_growing/label_connected_comp.py

```diff
@@ -55,7 +55,10 @@ def _convert_input_cloud(self, points):
         # python land)
         xs = (points[self.mask, 0]).astype(pycc.PointCoordinateType)
         ys = (points[self.mask, 1]).astype(pycc.PointCoordinateType)
-        zs = (points[self.mask, 2]).astype(pycc.PointCoordinateType)
+        if points.shape[1] == 2:
```

Comment:
Wist niet dat dit voor kan komen?!

Reply:
Was eerst ook niet zo, maar nu wil ik hem ook gebruiken voor clustering in 2D :)

Reply:
Haha ik dacht al :P